### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by cocojoe**
Made some changes around:
- Clarity of the user's iOS app vs their Auth0 app
- Removing magic variable squiggly brackets 

Fixed: Instance of Manage URL was using Docs magic variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,16 +95,16 @@ In your application bundle add a `plist` file named `Auth0.plist` with the follo
 <plist version="1.0">
 <dict>
   <key>ClientId</key>
-  <string>{YOUR_CLIENT_ID}</string>
+  <string>YOUR_AUTH0_CLIENT_ID</string>
```
Comment (albertoperdomo) [inline README.md]: Is this not breaking variable replacement in samples?

File: README.md [hunk 2]
```diff
@@ -122,21 +120,21 @@ In your application's `Info.plist` file, register your iOS Bundle Identifer as a
         <string>auth0</string>
         <key>CFBundleURLSchemes</key>
         <array>
-            <string>$(YOUR_BUNDLE_IDENTIFIER)</string>
+            <string>YOUR_BUNDLE_IDENTIFIER</string>
```
Comment (albertoperdomo) [inline README.md]: This the right syntax?

File: README.md [hunk 3]
```diff
@@ -122,21 +120,21 @@ In your application's `Info.plist` file, register your iOS Bundle Identifer as a
...
 ```
 
 > If your `Info.plist` is not shown in this format, you can **Right Click** on `Info.plist` in Xcode and then select **Open As / Source Code**.
 
-Finally, go to your [Auth0 Dashboard](${manage_url}/#/applications/${account.clientId}/settings) and make sure that **Allowed Callback URLs** contains the following entry:
+Finally, go to your [Auth0 Dashboard](https://manage.auth0.com) and make sure that your application's **Allowed Callback URLs** field contains the following entry:
 
 ```text
-{YOUR_BUNDLE_IDENTIFIER}://${YOUR_AUTH0_DOMAIN}/ios/{YOUR_BUNDLE_IDENTIFIER}/callback
+YOUR_BUNDLE_IDENTIFIER://YOUR_AUTH0_DOMAIN/ios/YOUR_BUNDLE_IDENTIFIER/callback
```
Comment (albertoperdomo) [inline README.md]: This the right syntax?

File: README.md (hunk 1 above)
Comment (cocojoe) [reply README.md]: This is just a README, if you are a developer you're not going to be relying upon variable replacement. The above is done inside the `auth0.plist` inside the auth0-sample.

File: README.md (hunk 2 above)
Comment (cocojoe) [reply README.md]: Yeah, not sure why it had $() tbh

File: README.md (hunk 3 above)
Comment (cocojoe) [reply README.md]: Yeah, no need for the extra curlys, and the $ bit makes less sense. You can see a practical example below @ https://github.com/auth0/Auth0.swift/pull/303/files/e46ed9886da18f8ddfd4279ed84508466a16c9da#diff-04c6e90faac2675aa89e2176d2eec7d8R140

**Comment by albertoperdomo** [comment]
LGTM!

**Pull request merged**